pkg/controller/pxc/controller.go

```diff
@@ -603,6 +604,11 @@ func (r *ReconcilePerconaXtraDBCluster) deploy(cr *api.PerconaXtraDBCluster) err
 	err = r.client.Create(context.TODO(), nodeSet)
 	if err != nil && !k8serrors.IsAlreadyExists(err) {
 		return errors.Wrap(err, "create newStatefulSetNode")
+	} else if k8serrors.IsAlreadyExists(err) {
+		err := r.createOrUpdate(nodeSet)
+		if err != nil {
+			return errors.Wrapf(err, "failed to update statefulset %s", nodeSet.Name)
+		}
 	}
```

Comment:
maybe it can be like this
```suggestion
	err := r.createOrUpdate(nodeSet)
	if err != nil {
		return errors.Wrap(err, "create newStatefulSetNode")
	}
```